doc/user_guide/_static/theme_overrides.css

```diff
@@ -0,0 +1,16 @@
+/* from https://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html */
+
+/* override table width restrictions */
```

Comment:
Please could you extend this comment a little to explain what this override is for (in case the above web page disappears).

Reply:
Note that this is a copy of `doc/developer_guide/_static/theme_overrides.css` so I updated both files.

Reply:
Great, thanks.